src/main/java/com/example/librarymanagement/dto/JournalDto.java

```diff
@@ -0,0 +1,32 @@
+package com.example.librarymanagement.dto;
+
+import com.example.librarymanagement.model.entity.Journal;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class JournalDto {
+    private LocalDate dateOfBorrowing;
+    private LocalDate dateOfReturning;
+    private String title;
+    private String authorName;
```

Comment:
does it make sense to return full dto of the author and library as it for the user?